bauble/src/convert.rs

```diff
@@ -522,6 +550,52 @@ impl<'a, T: FromBauble<'a>, const N: usize> FromBauble<'a> for [T; N] {
     }
 }
 
+#[cfg(feature = "enumset")]
+impl<'a, A, T> FromBauble<'a, A> for enumset::EnumSet<T>
+where
+    A: BaubleAllocator<'a>,
+    T: FromBauble<'a, A> + enumset::EnumSetType,
+{
+    const INFO: TypeInfo<'static> = TypeInfo::Kind(ValueKind::Array);
+
+    fn from_bauble(val: Val, allocator: &A) -> Result<A::Out<Self>, Box<DeserializeError>> {
+        match_val!(
+            val,
+            /*
+            // TODO: Parse bitflags.
+            (BitFlags(ty, flags), span) => {
```

Comment:
Will the
```rs
    const INFO: TypeInfo<'static> = TypeInfo::Kind(ValueKind::Array);
```
need to be adjusted when this is done?

Reply:
Yes, it would be changed to `ValueKind::BitFlags`